tests/test_dag_extension.py

```diff
@@ -0,0 +1,104 @@
+from marquez.airflow import MarquezDag
+from unittest.mock import Mock, patch, create_autospec
+from datetime import datetime
+from airflow.utils.state import State
+import airflow.models
+import pytest
+from croniter import croniter
+import pendulum
+import marquez.utils
+from marquez_client.marquez import MarquezClient
+
+
+class Context:
+    location = 'github://test_dag_location'
+    dag_id = 'test-dag-1'
+    namespace = 'test-namespace-1'
+    data_inputs = ["s3://data_input_1", "s3://data_input_2"]
+    data_outputs = ["s3://some_output_data"]
+    owner = 'test_owner'
+    description = 'this is a test DAG'
+    airflow_run_id = 'airflow_run_id_123456'
+    mqz_run_id = '71d29487-0b54-4ae1-9295-efd87f190c57'
+    start_date = datetime(2019, 1, 31, 0, 0, 0)
+    execution_date = datetime(2019, 2, 2, 0, 0, 0)
+    schedule_interval = '*/10 * * * *'
+
+    dag = None
+
+    def __init__(self):
+        self.dag = MarquezDag(
+            self.dag_id,
+            schedule_interval=self.schedule_interval,
```

Comment:
nit: please make sure that the formatting meets the `pep8` guidelines.

Reply:
there's a lot of stuff in pep8, so the easier route would be to turn on a python linter in circle CI (pylint or flake8).

- https://www.pylint.org/
- https://stackoverflow.com/questions/29469001/how-to-run-the-flake8-in-circle-yml

Reply:
^^ Don't worry about this now, created #4 to track it in the future